### PR TITLE
Only one between pec and SDI code must be mandatory

### PR DESCRIPTION
**Pull request by fabianaromagnoli**
Solve issue #3

### CODE REVIEW

**Review by lruozzi9 — CHANGES_REQUESTED** [review]
While doing the review I also thought of another situation. If I'm not wrong, the Italian SDI system accepts the code '0000000' as a valid SDI code. By entering this code, however, the PEC address must be present. I think we should add this case as well.

**Comment by mmenozzi** [comment]
We have to fix the Roave BC check...

**Review by lruozzi9 — APPROVED** [review]

File: src/Resources/translations/validators+intl-icu.it.yaml [hunk 1]
```diff
@@ -9,8 +9,10 @@ app:
         italian_sdi_code:
             valid: 'Per favore, inserisci un codice destinatario SDI valido.'
             not_blank: 'Il codice destinatario SDI non dovrebbe essere vuoto.'
+            required_if_pec_is_null: 'Per favore, inserisci almeno uno tra codice SDI e indirizzo PEC'
         pec_address:
             valid: 'Per favore, inserisci un indirizzo PEC valido.'
+            required_if_sdi_is_null: 'Per favore, inserisci almeno uno tra codice SDI e indirizzo PEC'
```
Comment (mmenozzi) [inline src/Resources/translations/validators+intl-icu.it.yaml:15]: I'd use the same message key.

File: src/Resources/translations/validators+intl-icu.en.yaml [hunk 2]
```diff
@@ -13,8 +13,10 @@ webgriffe_sylius_italian_invoiceable_order:
         italian_sdi_code:
             valid: 'Please enter a valid SDI code.'
             not_blank: 'SDI code should not be blank.'
+            required_if_pec_is_null: 'At least one between PEC address and SDI code is required'
         pec_address:
             valid: 'Please enter a valid PEC address.'
+            required_if_sdi_is_null: 'At least one between PEC address and SDI code is required'
```
Comment (mmenozzi) [inline src/Resources/translations/validators+intl-icu.en.yaml:19]: I'd use the same message key

**Review by mmenozzi — APPROVED** [review]

**Review by lruozzi9 — APPROVED** [review]

**Pull request merged**